user-interface/frontend/themes/datamanager/components/navigation-drawer.css

```diff
@@ -22,6 +22,14 @@
   color: var(--lumo-primary-color);
 }
 
+.project-navigation-drawer vaadin-side-nav-item.experiment-header {
+  color: var(--lumo-primary-color);
+}
+
+.exp-create-icon {
+  cursor: pointer;
+}
```

Comment:
can be replaced with the "clickable" classname for which there is this css already
```suggestion
```